docs/source/getting_started/install.rst

```diff
@@ -1,26 +1,39 @@
 Installation
 ==============
 
-Install from conda:
+For most users we recommend a conda installation:
 
 .. code:: bash
 
-    conda install -c conda-forge -c omnia -c mosdef foyer
+    conda install -c conda-forge -c omnia foyer
```

Comment:
do we still need omnia? (I think we might for windows, so it's fine if we use this as the most general case.)

Reply:
Can someone confirm that we still need `omnia` for windows?

Reply:
Yeah, I remember when I removed the omnia from our azure pipeline, all the windows build failed.